flow/data_pipeline/query.py

```diff
@@ -594,3 +599,30 @@ class QueryStrings(Enum):
             AND agg.baseline_source_id = baseline.source_id
         ORDER BY agg.submission_date, agg.submission_time ASC
         ;"""
+
+    FACT_TOP_SCORES = """
+        WITH curr_max AS (
+            SELECT
+                network,
+                submission_date,
```

Comment:
this should be `date AS submission_date` since `leaderboard_chart_agg` don't have a `submission_date` column